lib/metrics.py

```diff
@@ -101,20 +101,18 @@ def __append(self, func, iterable):
   def __get(self):
     self.__collect_async()
     result = list(itertools.chain(*self.results))
-    return map(
-      lambda v: round(v, 4), (
+    return [
```

Comment:
We will lose the rounding that we desire.  Please just wrap the previous code with a `list()` cast.

Reply:
how about use float(round(v, 4)) for v in (

Reply:
did the change